### PR TITLE
Tidy up

### PR DESCRIPTION
**Pull request by JessyCatterwaul**
Assorted minor cleanup 🧼

### CODE REVIEW

**Comment by JessyCatterwaul** [comment]
Dependent on #475

File: Emitron/Emitron/UI/Shared/Content Detail/ChildContentListingView.swift
Comment (0xTim) [inline Emitron/Emitron/UI/Shared/Content Detail/ChildContentListingView.swift]: How does this fit the with the RW style guide? This change looks ambiguous and less clear to me than the original

**Review by 0xTim — COMMENTED** [review]
LGTM assuming these changes work on Xcode 11, otherwise we'll need to wait until Xcode 12 is release. One comment regarding readability

File: Emitron/Emitron/UI/Shared/Content Detail/ChildContentListingView.swift
Comment (JessyCatterwaul) [reply Emitron/Emitron/UI/Shared/Content Detail/ChildContentListingView.swift]: It's not in the style guide; I don't think anybody knows that you can put trailing closures after labels. Otherwise I don't think this [thread on renaming](https://forums.swift.org/t/renaming-trailing-closure-functions-in-the-standard-library/35454) would have come about. (It's a pitch that would change, among other things, `sorted` to `sortedBy`. But the argument falls flat for me because, as you see in this commit, you can use `by`, `where`, `while`, etc., with a trailing closure. The label just needs to be within the parentheses. Not that that is not the same thing as labeled trailing closures like in Swift 5.3—instead, it's taking advantage of being able to refer to methods with arguments as closures, paired with trailing closure syntax.)

Personally, I don't feel the `by` is helpful, so I'd be inclined to just get rid of it. But because it was in the original code, I figured somebody liked it enough to not make it disappear.

Comment (sammyd) [reply Emitron/Emitron/UI/Shared/Content Detail/ChildContentListingView.swift]: I've removed `(by:)`. I think it still makes sense.

**Review by sammyd — APPROVED** [review]
:+1: LGTM

**Pull request merged**
